e2etests/cypress.config.js

```diff
@@ -26,7 +26,10 @@ module.exports = defineConfig({
 
   e2e: {
 
-    baseUrl: 'https://google.co.in/',
+    baseUrl: 'https://piro.matr.io/',
+    username: 'tri.jaramillo.group@tri.global',
+    password: 'PiroTest1234!',
```

Comment:
credentials should be from secrets

Reply:
Given that we've exposed the password in the git history, we should probably create a new test user account, unfortunately we can't use this username/e-mail because it's no longer active, so I'll create another.

Reply:
Does anyone have access to github secrets? I dont see the "settings" enabled for me. If so we can move this password to github secrets

Reply:
secrets are removed from script to github secrets.